design-proposals/README.md

```diff
@@ -86,5 +86,6 @@ Next KLIP number: **47**
 | [KLIP-43: TIMESTAMP data type support ](klip-43-timestamp-data-type-support.md)                                                  | Approved |                   |            | [Discussion](https://github.com/confluentinc/ksql/pull/6649) |
 | [KLIP-44: ksqlDB Go client](klip-44-ksqldb-golang-client.md)                                                                     | Proposal |                   |            | [Discussion](https://github.com/confluentinc/ksql/pull/6825) |
 | [KLIP-45: Materializing Tables for Query](klip-45-materializing-tables-for-query.md)                                             | Proposal |                   |            | [Discussion](https://github.com/confluentinc/ksql/pull/6997) |
-| KLIP-46: DATE and TIME data type support                                                                                         | Proposal |        0.20.0     |            | [Discussion](https://github.com/confluentinc/ksql/pull/7417) |
+| [KLIP-46: DATE and TIME data type support](klip-46-date-and-time-data-type-support.md)                                           | Proposal |      0.20.0       |            | [Discussion](https://github.com/confluentinc/ksql/pull/7417) |
+| [KLIP-47: Add source stream/table semantic](klip-47-add-source-stream-table-semantic.md)                                         | Proposal |                   |            | [Discussion](https://github.com/confluentinc/ksql/pull/7474) |
```

Comment:
KLIP 47 is already taken. Follow the proposal intent procedure to lock in a number before opening the KLIP: https://github.com/confluentinc/ksql/blob/master/design-proposals/README.md#proposal-intent

Reply:
Sounds good, created PR: https://github.com/confluentinc/ksql/pull/7496/

Reply:
🐎